examples/example6.c++

```diff
@@ -0,0 +1,35 @@
+#define NONIUS_RUNNER
+#include <nonius.h++>
+
+#include <string>
+#include <vector>
+
+NONIUS_PARAM("size", 100u)
+NONIUS_PARAM("string", std::string{"foo"})
+
+NONIUS_BENCHMARK("push_back vector", [](nonius::chronometer meter)
+{
+    auto n = meter.param("size");
+    auto s = meter.param<std::string>("string");
```

Comment:
While I don't mind the stringly typed parameters too much, someone made me aware of a technique that can use the compiler to ensure type-safety by using type tags instead of strings for the keys. I'll try it out and if it works I'll make that change before merging this.